tests/mixins/responsiveManagerMixin-test.js

```diff
@@ -0,0 +1,63 @@
+jest.dontMock('../../js/mixins/responsiveManagerMixin');
+jest.dontMock('../../js/components/utils');
+
+const React = require('react');
+const Enzyme = require('enzyme');
+const debounce = require('lodash.debounce');
+const createReactClass = require('create-react-class');
+const responsiveManagerMixin = require('../../js/mixins/responsiveManagerMixin');
+
+let Foo = createReactClass({
```

Comment:
I believe you won't reassign Foo. So it's a const.

Reply:
I thing it may be changeable too.